lib/row_transformer.rb

```diff
@@ -50,7 +42,7 @@ def transform
     end
 
     def transform_row
-        @transformed_row = @@box_columns.zip(@db_row).to_h
+        @transformed_row = @db_row
     end
```

Comment:
I think this method can be eliminated and either `@transformed_row` be refactored away or just have this set in `initialize`